src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/SettleClaimCallbackHandler.java

```diff
@@ -45,26 +44,30 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
-    private CallbackResponse checkState(CallbackParams callbackParams) {
-        CaseData caseData = callbackParams.getCaseData();
-        List<String> errors = new ArrayList<>();
-
-        SettleClaimHelper.checkState(caseData, errors);
-
-        return AboutToStartOrSubmitCallbackResponse.builder()
-            .errors(errors)
-            .build();
-    }
-
     private CallbackResponse saveJudgmentPaidInFullDetails(CallbackParams callbackParams) {
         CaseData.CaseDataBuilder<?, ?> caseDataBuilder = callbackParams.getCaseData().toBuilder();
         caseDataBuilder.previousCCDState(callbackParams.getCaseData().getCcdState());
+        String authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
+
+        deleteMainCaseDashboardNotifications(caseDataBuilder, authToken);
+
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(caseDataBuilder.build().toMap(objectMapper))
             .state(CASE_SETTLED.name())
             .build();
     }
 
+    private void deleteMainCaseDashboardNotifications(CaseData.CaseDataBuilder<?, ?> caseDataBuilder, String authToken) {
+        if (caseDataBuilder.build().isApplicantLiP()) {
```

Comment:
Please change this - avoid using dashboard client.

Reply:
done